prow.sh

```diff
@@ -1254,7 +1257,8 @@ main () {
         fi
 
         if tests_need_non_alpha_cluster; then
-            start_cluster || die "starting the non-alpha cluster failed"
+	    # Need to (re)create the cluster.
+            start_cluster "${CSI_PROW_E2E_GATES}" || die "starting the non-alpha cluster failed"
```

Comment:
Indention issue (tab vs. spaces)...

```suggestion
            # Need to (re)create the cluster.
            start_cluster "${CSI_PROW_E2E_GATES}" || die "starting the non-alpha cluster failed"
```